backend/WebApi/WebApiKernel.cs

```diff
@@ -7,9 +7,10 @@ public static class WebApiKernel
     public static void Setup(IServiceCollection services)
     {
         services.AddHttpClient();
-        services.AddSingleton<ProjectService>();
+        services.AddScoped<ProjectService>();
```

Comment:
This is now scoped, because it's using the project-context, which is obviously also scoped. Other options:
1) We inject the project-context into the controller
2) We just pull the project-code out of the header as an action parameter

Reply:
I'm fine with our services being scoped. It's our db context where it's important to get it right.